easybuild/easyconfigs/d/darshan-runtime/darshan-runtime-3.4.4-gompi-2023a-HDF5-1.14.0.eb

```diff
@@ -0,0 +1,50 @@
+easyblock = 'ConfigureMake'
+
+name = 'darshan-runtime'
+version = '3.4.4'
+versionsuffix = '-HDF5-1.14.0'
```

Comment:
Any particular reason to add the suffix here? It's not needed and we typically don't add one when there is just one version of the dependency in the toolchain anyway.

Reply:
Done in e24dca74bf23a2496eaeeea3f4a6361137286759